_data/simple-icons.json

```
@@ -15020,6 +15020,11 @@
                 "url": "https://aka.ms/fluentui-assets-license"
             }
         },
+        {
+            "title": "Yandex Cloud",
+            "hex": "5282FF",
+            "source": "https://cloud.yandex.com/en/brandbook"
```

```suggestion
            "source": "https://cloud.yandex.com/en/brandbook",
            "guidelines": "https://cloud.yandex.com/en/brandbook"
```
Since the guidelines are also given on the same page, it would be good to add this to the JSON data.